components/tools/OmeroPy/src/omero/plugins/obj.py

```diff
@@ -557,6 +557,7 @@ def process(self, args):
                     line = line.strip()
                     if line and not line.startswith("#"):
                         actions.append(self.parse(state, line=line))
+                fileinput.close()
```

Comment:
Perhaps in a try/finally block? I looked for best-practices. Perhaps cF. http://www.zopatista.com/python/2013/11/26/inplace-file-rewriting/